.github/workflows/ci-pre-commit.yml

```diff
@@ -28,6 +28,10 @@ jobs:
           profile: minimal
           toolchain: nightly-2024-04-10
           components: rustfmt, clippy
+      - name: Install Protoc
+        uses: arduino/setup-protoc@v3
```

Comment:
why do we need this

Reply:
litesvm dependency in turns has a bunch of solana dependencies that need this